ebu_tt_live/xsd1.1/ebutt_datatypes.xsd

```diff
@@ -65,7 +65,7 @@ Please note that the EBU-TT XML Schema is a helping document and NOT normative b
 	<xs:simpleType name="paddingType">
 		<xs:restriction base="xs:token">
 			<xs:pattern
-				value="(\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))|([+-]?\d*\.?\d+(px|c|%))\s([+-]?\d*\.?\d+(px|c|%))"
+				value="((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%)))(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*(\s((\d*(\.\d+)?(px|c|%))|([+-]?\d*(\.\d+)?(px|c|%))))*"
```

Comment:
No for each of the for values you have to possibilities, separated by "|", one with the [+-], one without. The four values are separated by the "\s".